thoth/messaging/advise_justification.py

```diff
@@ -29,14 +29,14 @@ class AdviseJustificationMessage(MessageBase):
     """Class used for Advise justification events on Kafka topic."""
 
     topic_name = "thoth.advise-reporter.advise-justification"
-    _message_version = 1  # update on schema change
 
     class MessageContents(BaseMessageContents, serializer="json"):  # type: ignore
         """Class used to represent contents of advise justification message Kafka topic."""
 
         message: str
         justification_type: str
         count: int
+        version: str = "v1"
```

Comment:
How do we guarantee investigator can handle v1 and v2 at the same time using same version of thoth-messaging?

Reply:
https://github.com/thoth-station/investigator/pull/300

Reply:
But if I update thoth-messaging I cannot look back to v1 if we moved to v2 of messages right?